api/reference/services.md

```diff
@@ -702,18 +699,15 @@ Or via the delete action for deleting multiple services:
 }
 ```
```

Comment:
This is "broken" in github highlighting because the json above here has a `...` which is invalid json.  However, it renders correctly in reality.